weblate/addons/base.py

```diff
@@ -116,12 +116,14 @@ def configure(self, settings):
 
     def post_configure(self, run: bool = True):
         component = self.instance.component
+        project = self.instance.project
 
         # Configure events to current status
         component.log_debug("configuring events for %s add-on", self.name)
+        project.log_debug("configuring events for %s add-on", self.name)
```

Comment:
These can be None, so something like this is needed:

```suggestion
        (component or project).log_debug("configuring events for %s add-on", self.name)
```